src/classes/PS_CommitmentRequest.cls

```diff
@@ -366,6 +389,14 @@ public inherited sharing class PS_CommitmentRequest {
             paymentMethodType = PaymentMethod.CARD.name();
         }
 
+        /***
+        * @description Set boolean to de activate all existing schedule
+        * @return RequestBody This request body instance 
+        */
+        public RequestBody deactivateOldSchedules() {
+            this.deactivateAllExistingSchedules = true;
+            return this;
+        }
```

Comment:
Can we include an empty line between this and the beginning of the following description?